packages/react/src/primitives/PhoneNumberField/PhoneNumberField.tsx

```diff
@@ -56,6 +88,9 @@ const PhoneNumberFieldPrimitive: Primitive<PhoneNumberFieldProps, 'input'> = (
       isDisabled={isDisabled}
       isReadOnly={isReadOnly}
       onInput={onInput}
+      // Pass the onChange function to the TextField and use the formattedPhoneNumber state as the value
+      onChange={onNumberChange}
+      value={formattedPhoneNumber}
```

Comment:
Also think this handling breaks controlled usage of the component since we are ignoring `value`?

Reply:
I have a logic that removes all non digit to avoid any symbols gets entered and there is `+1` already included in the dropdown of dial codes, so if you enter `8001231212` it will be formatted to `(800) 123-1212`

Reply:
Say I currently have a form in my application that uses a `PhoneNumberField`, using the example values above the value of the field submitted to the form would go from `8001231212` to `(800) 123-1212` which is a breaking change right?

Reply:
Yes, I think so, any formatting of the `8001231212` would be a breaking change from the data that gets submitted with a form perspective